Bottom: 0584d5ae66e14fe1403022d20c852784c9848bb6
Top:    e5653d2b6b78091eee714f50d34c07e1529fbc95
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-31 00:27:25 +0100

Refresh of move-more-rendering-to-world-c

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 2df5ea5..0869215 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1953,20 +1953,8 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
 
   hidgl_init_triangle_array (&buffer);
   ghid_invalidate_current_gc ();
-<<<<<<< current
-  glPushMatrix ();
-  glScalef ((ghid_flip_x ? -1. : 1.) / port->zoom,
-            (ghid_flip_y ? -1. : 1.) / port->zoom, 1);
-  glTranslatef (ghid_flip_x ? port->view_x0 - PCB->MaxWidth  :
-                             -port->view_x0,
-                ghid_flip_y ? port->view_y0 - PCB->MaxHeight :
-                             -port->view_y0, 0);
   DrawAttached ();
   DrawMark ();
-=======
-  DrawAttached (TRUE);
-  DrawMark (TRUE);
->>>>>>> patched
   hidgl_flush_triangles (&buffer);
 
   ghid_show_crosshair (TRUE);
